src/main/java/com/powsybl/openloadflow/network/LfNetworkParameters.java

```diff
@@ -28,119 +28,95 @@ public class LfNetworkParameters {
 
     public static final double MAX_PLAUSIBLE_TARGET_VOLTAGE_DEFAULT_VALUE = 1.2;
 
-    private SlackBusSelector slackBusSelector;
+    private SlackBusSelector slackBusSelector = new FirstSlackBusSelector();
 
-    private final GraphConnectivityFactory<LfBus, LfBranch> connectivityFactory;
+    private GraphConnectivityFactory<LfBus, LfBranch> connectivityFactory = new EvenShiloachGraphDecrementalConnectivityFactory<>();
 
-    private final boolean generatorVoltageRemoteControl;
+    private boolean generatorVoltageRemoteControl = false;
```

Comment:
This is a weird default value...

Reply:
this is historical reason... I guess it was because of we didn't want to change too many unit tests?